ocaml/xapi/xapi.ml

```diff
@@ -783,6 +783,21 @@ let set_stunnel_timeout () =
     Stunnel.timeoutidle := Some timeout
   with _ -> debug "Using default stunnel timeout (usually 43200)"
 
+let init_tls_verification ~__context () =
+  (* set global flag based on database entry *)
+  let file = Xapi_globs.disable_tls_verification_path in
+  match Sys.file_exists file with
+  | true ->
+      warn "TLS verification is disabled on this host: %s" file ;
+      Stunnel_client.set_verify_by_default false
+  | false ->
+      let pool = Helpers.get_pool ~__context in
+      let default =
+        Db.Pool.get_tls_verification_enabled ~__context ~self:pool
```

Comment:
I mentioned this yesterday - I don't think this will work https://github.com/xapi-project/xen-api/pull/4351#discussion_r599630590

In order to decide whether to verify tls connections, slaves would have to connect to the master, but how do they know whether to verify that connection?

If you change `Xapi_globs.disable_tls_verification_path` to instead be `Xapi_globs.enable_tls_verification_path`, then the problem goes away